ts/html/index.ts

```diff
@@ -0,0 +1,12 @@
+import linkTextInternal from '../../js/modules/link_text';
+
+
+export const linkText = (value: string): string =>
+  linkTextInternal(value, { target: '_blank' });
+
+export const replaceLineBreaks = (value: string): string =>
+  value.replace(/\r?\n/g, '<br>');
+
+// NOTE: How can we use `lodash/fp` `compose` with type checking?
+export const render = (value: string): string =>
+  replaceLineBreaks(linkText(value));
```

Comment:
Would have been nice to do `compose(replaceLineBreaks, linkText)` but I couldn’t get it to type check with TypeScript. Didn’t spend much time on it but will revisit it later. This article alludes to this: https://www.reaktor.com/blog/fear-trust-and-purescript/

Reply:
That is one of the most frustrating things about typescript and anything that tries to add types to JavaScript. Natural things in JavaScript, standard things from the toolbelt, result in unexpected pain. Sounds like you weren't cool with coercing to `any` here to make it work?

Reply:
Agreed, this is frustrating. But it’s not inherent to the idea of types, just the limitations of the TypeScript type system. Examples like this make me interested in more powerful (but not necessarily more complex) type systems present in languages such as PureScript. It uses the Hindley–Milner type system which is decades old, well researched and understood, and which is able to infer basically all types automatically without requiring manual type signatures unless desired for documentation.

I have a feeling even for this one there, there might be a way given the `compose` type definition in @DefinitelyTyped: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/81d0e4984d4b5e10668020eacd4b9a4542c5c837/types/lodash/fp/compose.d.ts#L8-L15

I will revisit this again in the future.

Reply:
I hope this doesn’t discourage you and using `any` is a fair workaround, given that it’s the basic escape hatch that brings us back to untyped JavaScript 😄